lang/c++/test/CodecTests.cc

```diff
@@ -1264,6 +1264,46 @@ static const TestData3 data3[] = {
 
     {R"(["boolean", "int"])", "U1I", R"(["boolean", "long"])", "U1L", 1},
     {R"(["boolean", "int"])", "U1I", R"(["long", "boolean"])", "U0L", 1},
+
+    // Aliases
+    {"{\"type\":\"record\", \"name\":\"r\", \"fields\":["
```

Comment:
Could you use raw string literals to improve readability of the test?

Reply:
> Could you use raw string literals to improve readability of the test?

I am going to merge the PR!
@mkmkme You could make this improvement in a follow-up!

Reply:
Yup, will do!